buildmaster/NMCPD/kinematics.json

```diff
@@ -0,0 +1 @@
18,0.18,0.18,0.18,0.18,0.18,0.18,0.18,0.18,0.18,0.18,0.18,0.225,0.225,0.225,0.225,0.225,0.225,0.225,0.225,0.225,0.225,0.225,0.225,0.225,0.225,0.275,0.275,0.275,0.275,0.275,0.275,0.275,0.275,0.275,0.275,0.275,0.275,0.275,0.275,0.35,0.35,0.35,0.35,0.35,0.35,0.35,0.35,0.35,0.35,0.35,0.35,0.35,0.45,0.45,0.45,0.45,0.45,0.45,0.45,0.45,0.45,0.45,0.45,0.45,0.55,0.55,0.55,0.55,0.55,0.55,0.55,0.55,0.55,0.55,0.55,0.675,0.675,0.675,0.675,0.675,0.675,0.675,0.675,0.675,0.675	]},{"Q2":[0.16,0.25,0.35,0.45,0.6,0.17,0.25,0.35,0.45,0.63,0.88,1.12,0.16,0.25,0.35,0.45,0.61,0.88,1.13,1.38,1.71,0.16,0.25,0.35,0.45,0.64,0.86,1.12,1.37,1.75,2.24,2.73,3.46,0.16,0.26,0.35,0.45,0.62,0.88,1.12,1.37,1.74,2.23,2.74,3.46,4.47,5.41,0.25,0.35,0.45,0.62,0.88,1.12,1.37,1.75,2.24,2.73,3.48,4.47,5.49,6.83,0.26,0.35,0.45,0.62,0.86,1.13,1.37,1.74,2.24,2.74,3.45,4.47,5.48,6.92,8.92,0.36,0.45,0.64,0.86,1.13,1.38,1.74,2.24,2.74,3.46,4.45,5.47,6.92,8.96,11.45,14.36,0.46,0.61,0.88,1.13,1.37,1.74,2.25,2.74,3.46,4.46,5.46,6.9,8.93,11.44,14.82,19.19,0.68,0.86,1.11,1.38,1.74,2.24,2.75,3.47,4.47,5.47,6.91,8.91,11.4,14.89,19.63,26.07,0.9,1.11,1.38,1.76,2.24,2.75,3.49,4.47,5.46,6.91,8.92,11.37,14.87,19.74,26.36,34.74,1.13,1.38,1.75,2.24,2.74,3.49,4.47,5.46,6.9,8.92,11.37,14.85,19.74,26.52,35.32,44.94,1.4,1.75,2.24,2.74,3.47,4.48,5.47,6.9,8.92,11.37,14.84,19.76,26.55,35.28,46.95,1.82,2.24,2.75,3.47,4.47,5.49,6.92,8.93,11.37,14.85,19.75,26.61,35.37,47.01,63.04,2.28,2.74,3.48,4.47,5.47,6.97,8.93,11.38,14.85,19.74,26.64,35.42,46.95,63.23,2.78,3.46,4.47,5.47,6.89,8.94,11.37,14.87,19.74,26.6,35.43,46.98,63.48,90.68,3.57,4.51,5.48,6.9,8.91,11.43,14.85,19.74,26.63,35.45,47.12,63.52,96.35,4.54,5.47,6.93,8.92,11.34,14.89,19.77,26.64,35.5,47.26,63.65,98.05,5.53,6.88,8.91,11.34,14.85,19.74,26.64,35.57,47.16,63.56,98.82,7.04,8.88,11.36,14.85,19.79,26.49,35.4,47.03,63.53,99.03]},{"y":[0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]}]"kinematics_min":[{"x":null},{"Q2":null},{"y":null}]"kinematics_max":[{"x":null},{"Q2":null},{"y":null}]}
```

Comment:
Are these `null` expected? What should be returned if an user asks for the "minimums array", just the same thing as the avg?

Reply:
I placed a `null` because this is a case in which bin edges are not provided by the experimentalists; I'd say that if `null`, then min=avg.

Reply:
I see, thanks.
I also I think there might have a bug in the script that outputs the json because the `kinematics_max` and `kinematics_min` are missing a comma in front.

Also, I think `[{"x":null},{"Q2":null},{"y":null}]` could just be a single dictionary: `{"x":null, "Q2":null, "y":null}` (I'm using the `null` as example because the arrays are big, but the same applies.